recipes-tag/attrs/meta.yaml

```diff
@@ -22,16 +22,13 @@ requirements:
   run:
     - python
     - zope
+    - zope.interface
```

Comment:
do we really need these things at runtime?!

It looks like these are only testing or helper dependencies, see https://github.com/python-attrs/attrs/blob/master/setup.py